nautobot/extras/forms.py

```diff
@@ -1309,6 +1311,23 @@ class ScheduledJobFilterForm(BootstrapMixin, forms.Form):
     total_run_count = forms.IntegerField(required=False)
 
 
+#
+# Notes
```

Comment:
I don't see any REST API support for notes - at a minimum we probably need a dedicated `notes` endpoint similar to the existing `object-changes` endpoint, but it'd be a lot more user-friendly if we could easily retrieve notes associated with a specific object instance as well.

Reply:
This is going to be a separate issue. It would make sense to have a dedicated endpoint, but the Notes List view was not scoped. So, the API would have something the UI didn't.  Also, I don't mind getting this in and then turning around and trying to get the API in before 1.4 release happens.

Agreed on the second part, but not something I see any examples on in the current architecture for anything.

Reply:
I'm OK with tracking this as a follow-on issue as well.

Reply:
Issue #2106 created for tracking the API Support.  I am actively working on this, it is possible I will get the API support added before we release 1.4